static/sass/patterns/_icon-blocks.scss

```diff
@@ -0,0 +1,74 @@
+// Icon Blocks
+// ===========
+
+.icon-blocks-container {
+  @include center('medium-page');
+  @include grid;
+
+  h2 {
+    @supports not (display: grid) {
```

Comment:
I thought it was best to keep some of these inside the @supports not (display: grid) statement in order to not override them in the grid statement. I can rethink this, but since we have such low old IE traffic I don't know if it is worth it.